compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_instruction_gen.bal

```diff
@@ -541,6 +541,23 @@ type InstructionGenerator object {
         }
     }
 
+    function generateFPLoadIns(bir:FPLoad inst) {
+        string lambdaName = inst.name.value + "$lambda$";
```

Comment:
Shall we add this as constant. BTW why we need to concat "$lambda$" to instruction name?

Reply:
This is going to be the name of the generated java lambda function. We have ballerina lambda functions with the same name as well. Hence appended to make this unique. 

This is the only place this string is used, hence no point of moving to a constant.

Thanks